src/slices/AccountSlice.ts

```diff
@@ -176,17 +173,6 @@ export const getBalances = createAsyncThunk(
     } catch (e) {
       handleContractError(e);
     }
-
-    try {
-      const poolTokenContract = new ethers.Contract(
-        addresses[networkID].PT_TOKEN_ADDRESS as string,
-        ierc20Abi,
-        provider,
-      ) as IERC20;
-      poolBalance = await poolTokenContract.balanceOf(address);
-    } catch (e) {
-      handleContractError(e);
-    }
```

Comment:
these changes seem unrelated to bonds. Just wanted to make sure removal was intentional here

Reply:
yea, was more just tired of making those calls all the time when they are unnecessary.